### DISCUSSION TITLE
No syntax highlighting in .eex files?

### DISCUSSION THREAD

**Discussion by schrute99** [elixir-tools/discussions#4] [Q&A] [answered]
Hi, thanks for creating this awesome plugin!

I'm using the advanced setup from the readme with neovim v0.9.0-dev-920+gc9b0fe1f4 and template files that end with `.eex` don't have any syntax highlighting. Neovim detects the file type as "eelixir" (bottom right).

Is there any way to activate syntax highlighting for these files?

Cheers!

**Comment by mhanberg** [comment, marked as answer]
You will use treesitter for highlighting.
